packages/applications-service-api/prisma/schema.prisma

```diff
@@ -54,29 +54,31 @@ model Document {
 }
 
 model Representation {
-  id                    Int       @unique @default(autoincrement())
-  representationId      Int       @id
+  id                    Int          @unique @default(autoincrement())
+  representationId      Int          @id
   caseReference         String
   caseId                Int
   referenceId           String?
   status                String?
   dateReceived          DateTime?
-  representationComment String?   @db.NText
+  representationComment String?      @db.NText
   representationFrom    String?
   representationType    String?
   registerFor           String?
   representedId         String?
   representativeId      String?
+  representative        ServiceUser? @relation("representative", fields: [representativeId], references: [serviceUserId], onDelete: NoAction, onUpdate: NoAction)
```

Comment:
I don't think we want to add these relationships, they are covered with Id's abbove.

Reply:
I thought I had reverted this! Thanks for catching this.